Feature/ong4 63/screen listado categorias

DiegoSantelices10 opened the pull request:
Se realizo el screen para visualizar todas las categorías y tener la posibilidad de crear, editar y eliminar la misma.

![Screenshot_31](https://user-images.githubusercontent.com/70447941/202633649-a379c197-2722-4872-b1c7-759f2e35ae96.png)

SoraBeat committed:
Listado de usuarios(borrado/edicion/creacion) con paginacion y filtros

SoraBeat committed:
Skeleton y debounce a la consulta axios

SoraBeat committed:
Agregado del key a los skeleton

DiegoSantelices10 committed:
update

DiegoSantelices10 committed:
update

DiegoSantelices10 committed:
Merge branch 'feature/ONG4-65/ScreenListadoDeUsuarios(backoffice)' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-63/ScreenListadoCategorias

DiegoSantelices10 committed:
Creacion de listado de categorias

DiegoSantelices10 committed:
Listado de las categorias

SoraBeat reviewed (approved):
Esta funcional y cumple con los requisitos dados. Solo sacaria el filtro de "usuario"(el drop down list) si no es necesario. Tambien tener en cuenta los console log de prueba

milandre commented on src/Components/Categories/index.jsx:
```diff
@@ -0,0 +1,189 @@
+					name="pagination"
+					setOnChange={setAmountToShow}
+				/>
+				<TableDropDownList
```
Sacar este filtro que no corresponde a las Categorias

milandre commented on src/Components/Categories/index.jsx:
```diff
@@ -0,0 +1,189 @@
+										</TableFieldContainer>
+										<TableFieldContainer>
+											<Link to={"/backoffice/update-category/" + category.id}>
+												<FaRegEdit size={30} className=" text-yellow-500" />
```
Aca hay un icono y en el PR #33 es un boton. Sugiero ponerse de acuerdo en cual va a ser el estilo final para cada una de las tablas con respecto a estos botones

DiegoSantelices10 replied:
Vamos a dejar los iconos!

milandre commented on src/Components/Categories/index.jsx:
```diff
@@ -0,0 +1,189 @@
+										</TableFieldContainer>
+										<TableFieldContainer>
+											<button onClick={() => handleDeleteCategory(category.id)}>
+												<FaRegTrashAlt size={30} className="text-red-600" />
```
Aca hay un icono y en el PR https://github.com/alkemyTech/React-SXXI-T4/pull/33 es un boton. Sugiero ponerse de acuerdo en cual va a ser el estilo final para cada una de las tablas con respecto a estos botones

DiegoSantelices10 replied:
Vamos a dejar los iconos!

milandre reviewed (changes requested):
Dejo comentarios y sugiero leer tambien lo que dice @SoraBeat

DiegoSantelices10 committed:
Correcciones requeridas

Se realizaron los cambios que fueron requeridos y asi cumplir con lo asignado en la tarea.

DiegoSantelices10 committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-63/ScreenListadoCategorias
